Makefile

```diff
@@ -423,6 +423,12 @@ up-detach: build ## Bring up local Grapl and detach to return control to tty
 		--file docker-compose.yml \
 		up --detach --force-recreate --always-recreate-deps --renew-anon-volumes
 
+.PHONY: up-pulumi
+up-pulumi: export COMPOSE_PROJECT_NAME="grapl"
+up-pulumi:  ## launch pulumi via docker-compose up
+	$(WITH_LOCAL_GRAPL_ENV)
+	docker-compose -f docker-compose.yml up pulumi
```

Comment:
actually, we probably want a `docker-compose run` here instead of `up`; notably, it propagates the exit code of the `pulumi` run.

Reply:
Fair enough. I think the up may have as well, but really its a single run that we want to execute. The service doesn't stay up so run makes more sense